pkg/parser/testdata/proto-instance.yml

```diff
@@ -0,0 +1,22 @@
+aliases: &container_body
+  image: alpine:latest
+  platform: linux
+  privileged: true
+  cpu: 2.5
+  memory: 4G
+  additional_containers:
+    - name: mysql
+      image: mysql:latest
+      port: 3306
+      cpu: 1
+      memory: 1024
+      env:
+        MYSQL_ROOT_PASSWORD: ""
+
+regular_task:
+  container:
+    <<: *container_body
+
+proto_task:
```

Comment:
What's the difference between two of these tasks?

Reply:
One is parsed via the regular parser and the `proto` one with the new generic one. And they yield the same results!!!

Reply:
Besides the fact that `proto-instance.json` is parsed with the RPC parser and `proto-instance.yml` with the Go parser — these two tasks (`regular` and `proto`) look essentially the same and seem to be treated equally, why duplicate them?

Reply:
This is the Go YAML `parser` package. I can leave only `proto` task but in case there will be some changes in the regular YAML parser of `container` instances it will be great to see this test to fail and make sure that `proto_container` will keep in sync.